src/applications/personalization/profile/components/direct-deposit/BankInfoCNP.jsx

```diff
@@ -3,65 +3,66 @@ import PropTypes from 'prop-types';
 import { Prompt } from 'react-router-dom';
```

Comment:
Sorry about all the diff noise in this file. What _was_ BankInfoCNPv2 is now BankInfoCNP, so you can basically ignore any "changes" that are shown here.